projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2694,6 +2694,13 @@ public static function get_updateable_data_list( $selector = '' ) {
 				'validate_callback' => __CLASS__ . '::validate_boolean',
 				'jp_group'          => 'subscriptions',
 			),
+			'jetpack_verbum_subscription_modal'     => array(
+				'description'       => esc_html__( 'Show subscription pop-up after commenting on a post', 'jetpack' ),
+				'type'              => 'boolean',
+				'default'           => 0,
```

Comment:
In `get_option` the default value seems to be true, so need to check how this works.